test/unit/eventsTest.js

```diff
@@ -0,0 +1,144 @@
+const fs = require('fs');
+const path = require('path');
+const interpreted = fs.readFileSync(path.join(__dirname, '..', '..', 'src', 'p5.dance.interpreted.js'), 'utf8');
+const injectInterpreted = require('../helpers/injectInterpreted');
+
+const test = require('tape');
+const helpers = require('../helpers/createDanceAPI');
+
+async function runUserCode(userCode) {
+  const nativeAPI = await helpers.createDanceAPI();
+  return {
+    nativeAPI,
+    interpretedAPI: injectInterpreted(nativeAPI, interpreted, userCode),
+  };
+}
+
+test('atTimestamp adds a cue to the cue list', async t => {
+  const {nativeAPI, interpretedAPI} = await runUserCode(`
+    atTimestamp(2, "seconds", function () {});
+  `);
+
+  t.deepEqual(interpretedAPI.getCueList(), {seconds: [2], measures: []});
+
+  t.end();
+  nativeAPI.reset();
+});
+
+test('conflicting everySeconds and atTimestamp cues', async t => {
```

Comment:
These tests are great - I wonder if the coverage is sufficient here that we just remove the flakey spriteDanceTest.js?

Reply:
I'm tempted to keep both for now, because these new tests don't cover all the scenarios from the original tests.  Maybe we could do future cleanup to turn the integration tests (that use `runLevel.js`) into unit tests?